Documentation/ceph-csi-drivers.md

```diff
@@ -129,7 +139,7 @@ kubectl create -f cluster/examples/kubernetes/ceph/csi/example/rbd/storegeclass.
 
 ## Create RBD Secret
 
-Create a Secret that matches `adminid` or `userid` specified in the
+Create a Secret that will be used by the
```

Comment:
What would it take to get rid of this secret? The CSI driver should be able to look up a secret. Should we be generating the secret where the csi driver can find it, similar to the mons? (Of course this is independent from this PR)

Reply:
Yes. I have discussed that with @ShyamsundarR in the past and was planning on looking into this for a future PR. I need a litle feedback on the approach and planned on filing an issue as soon as I was done this pr.

Reply:
Issue #3387 filed.